src/Raven.Server/Documents/Indexes/Persistence/Lucene/LuceneIndexReadOperation.cs

```diff
@@ -88,15 +88,15 @@ public override long EntriesCount()
 
             // The reason why we keep distincts counts here, is because at the Lucene level there is no sorting unique accesse like we have 
             // in Corax with the SortingMatch. We have to know the page size to account for that. 
-            var pageSize = query.PageSize;
+            var pageSize = query.PageSize; //TODO stav: obsolete?
```

Comment:
?

Reply:
Skip and take in all sharded queries currently not working since the RewriteForPaging func in the ShardedQueryProcessor is setting the query's Offset and Limit. And here the entries are fetched according to Start and Pagesize, which are commented as obsolete (in their class). 
There is an issue: https://issues.hibernatingrhinos.com/issue/RavenDB-18998/Sharding-Queries-Handle-skip-and-take
Anyway, this comment can be removed I guess